tests/cubeprop/Da.cube.ref

```diff
@@ -77,7 +77,7 @@ Property: Da [e/a0^3]. Isocontour range for 85% of the density: (0.0697875,0)
  1.18433E-02  1.04189E-02  8.63150E-03  6.77687E-03  5.08580E-03  3.68099E-03 
  2.15389E-03  2.70243E-03  3.33736E-03  4.05961E-03  4.86617E-03  5.74235E-03 
  6.64919E-03  7.51158E-03  8.21632E-03  8.62834E-03  8.62836E-03  8.16425E-03 
- 7.28822E-03  6.14651E-03  4.92188E-03  3.76965E-03  2.78419E-03  1.41093E-03 
+ 7.28821E-03  6.14651E-03  4.92188E-03  3.76965E-03  2.78419E-03  1.41093E-03 
  1.76081E-03  2.16825E-03  2.63544E-03  3.16152E-03  3.73740E-03  4.33769E-03 
```

Comment:
Last time the numbers were way different (or maybe it was extra lines), so the close numbers here are reassuring. As is Ds unchanged.